packages/date-picker/src/DatePicker/components/Calendar/CalendarClasses.ts

```diff
@@ -1,30 +1,32 @@
+/* eslint-disable camelcase */
 import calendarStyles from "./Calendar.scss"
 
 export const defaultCalendarClasses = {
```

Comment:
```suggestion
import { ClassNames } from "react-day-picker"
import calendarStyles from "./Calendar.scss"

export const defaultCalendarClasses: ClassNames = {
```

Which actually surfaces that `interactionDisabled` is an invalid key